lib/beaker/host/pswindows.rb

```diff
@@ -23,5 +23,17 @@ def platform_defaults
       })
     end
 
+    attr_reader :scp_separator, :external_copy_base, :system_temp_path
+    def initialize name, host_hash, options
+      super
+
+      @scp_separator = '/'
+      # %TEMP% == C:\Users\ADMINI~1\AppData\Local\Temp
+      # is a user temp path, not the system path.  Also, it doesn't work, there's
+      # probably an issue with the `ADMINI~1` section
+      @system_temp_path = 'C:\\Windows\\Temp'
+      @external_copy_base = '/programdata'
```

Comment:
I've tried to use the `ALLUSERSPROFILE` command a number of times over this week, and it doesn't work in my testing.

The command returns `C:\PROGRA~3`, which is fine, but I can get the SCP call to match the working `/programdata` call, and it won't SCP correctly, giving me the message:

    localhost $ scp tmp/repo_configs/windows/puppet-agent-x64.msi win-2008r2-x86_64-agent:/PROGRA~3 {:ignore => }
    RuntimeError: scp: /PROGRA\~3: Error opening file: No such file or directory.

Regardless of whether the file(s) actually exist or not. I can verify that the directory does exist after the beaker run is complete (and before the command is called as well), and I've tried doing various things like creating the file before SCP-ing to it as a target.  I can even run this same command after a beaker run, and it'll work from my command line, but Beaker does not successfully make this call.

`/programdata` works in the cases that I've tested.  I'm confused about your point about Windows 2003.  When I run it by default, it uses the cygwin command and executes fine.  If I try to enable powershell by setting `is_cygwin: false` on the host, everything blows up, getting nowhere near running the command above.  I'm just using the host information below.  Is there something I'm missing to set this up correctly (like should I be pointing to a different template?)?

    windows2003r2-64-2:
      hypervisor: vmpooler
      platform: windows-2003r2-64
      template: win-2003r2-x86_64
      roles:
        - agent